scripts/deploy.js

```diff
@@ -42,12 +43,31 @@ const deploy = async (contractName, _args = [], overrides = {}, libraries = {})
   console.log(' 📄', chalk.cyan(contractName), 'deployed to:', chalk.magenta(deployed.address));
   console.log(' ⛽', chalk.grey(extraGasInfo), '\n');
 
+  if (network === 'localhost') {
+    networks.localhost.addresses[`${contractName}`] = String(deployed.address);
+  } else {
+    networks[`${network}`].addresses.AutoAppealableArbitrator = String(arbitratorAddress);
+    networks[`${network}`].addresses.SafientMain = String(deployed.address);
```

Comment:
Should we hardcode `SafientMain` or just use `contractName`?

Reply:
No hardcode is required, it is fetched from the `constants.js` file.